src/api/unified/symbol_manager.cpp

```diff
@@ -246,6 +246,14 @@ bool checkArray(af_backend activeBackend, af_array a) {
     return backend == activeBackend;
 }
 
+bool checkArray(af_backend activeBackend, const af_array* a) {
```

Comment:
I like this, it makes checking the output array (whether it exists yet or not) cleaner.